### PR TITLE
achievements & semesters sass

### PR DESCRIPTION
**Pull request by srenaxu**
@ahsu1230 @notewio

### CODE REVIEW

File: constellations/gemini-admin/src/achieve/achieve.sass [hunk 1]
```diff
@@ -47,4 +47,4 @@ a.add-achievement
     background-color: $colorTurquoise
     color: $colorWhite
     padding: 0.75em
-    border-radius: 10px
+    border-radius: 10px
```
Comment (notewio) [inline constellations/gemini-admin/src/achieve/achieve.sass:50]: everything looks fine, a couple of css ordering things though: padding & border before colors

File: constellations/gemini-admin/src/semester/semesterEdit.sass [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    & .buttons
+      margin-top: $spacing2x
+      margin-bottom: $spacing8x
+
+      & button
+        outline: none
+        border: none
+        border-radius: 6px
+        padding: $spacing2x $spacing4x
```
Comment (notewio) [inline constellations/gemini-admin/src/semester/semesterEdit.sass:35]: padding before borders

File: constellations/gemini-admin/src/semester/semester.sass [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  & h1
+    margin: $spacing4x 0
+    line-height: $lineHeight3
+    font-size: $fontSize3
+  & ul
+    padding: 0
+
+  & .semester-lists
+    list-style: none
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/semester/semester.sass:19]: list-style should be lowest priority here

**Comment by srenaxu** [comment]
@notewio can you please do one final review before i merge my changes

**Comment by notewio** [comment]
looks good to me

**Pull request merged**